app/qml/ActiveLayerPanel.qml

```diff
@@ -25,34 +25,25 @@ Drawer {
 
     id: layerPanel
     visible: false
-    modal: false
-    interactive: true
+    modal: true
+    interactive: false
     dragMargin: 0 // prevents opening the drawer by dragging.
 
     background: Rectangle {
         color: InputStyle.clrPanelMain
     }
 
-    Rectangle {
-        id: header
-        height: InputStyle.rowHeightHeader
-        width: parent.width
-        color: InputStyle.panelBackgroundLight
-
-        Text {
-            anchors.fill: parent
-            anchors.leftMargin: InputStyle.panelMargin
-            anchors.rightMargin: InputStyle.panelMargin
-            text: qsTr("Survey layer")
-            color: InputStyle.fontColor
-            font.pixelSize: InputStyle.fontPixelSizeTitle
-            font.bold: true
-            horizontalAlignment: Text.AlignLeft
-            verticalAlignment: Text.AlignVCenter
-        }
-
-        layer.enabled: true
-        layer.effect: Shadow {}
+    PanelHeader {
```

Comment:
Generally looks good, having swipe allowed on panel with list of items is not a good idea. However got two remarks:
- would be great if the panel closes on click outside as it is, so just the swiping will be disabled. Back button can be kept.
- Very similar panel is for MapThemes either with the same issue, would be nice to have them both as much similar as can be. Could you please update the MapThemePanel as well or at least create an issue? Many Thanks

Reply:
Thank you too for this great open source project. 
- I agree with you about the panel closes on click outside. Another problem is back button doesn't work when `interactive: false` I couldn't find a way for this. When interactive is true, clicking outside works but dragging down the layers area closes the panel too. `edge: Qt.LeftEdge` can work but I think it doesn't look better than `edge: Qt.BottomEdge` When `edge: Qt.LeftEdge` is active the panel is top of the screen.
- Yes I will test and update the pull request.
Thank you.

Reply:
I see, lets go with non-interactive panel to fix your problem. It seems ok now. Thank you!